libr/core/core.c

```diff
@@ -2201,8 +2201,8 @@ R_API bool r_core_init(RCore *core) {
 	core->print->exists_var = exists_var;
 	core->print->disasm = __disasm;
 	core->print->colorfor = (RPrintColorFor)r_core_anal_optype_colorfor;
-	core->print->hasrefs = (RPrintColorFor)r_core_anal_hasrefs;
-	core->print->get_comments = (RPrintCommentCallback) r_core_anal_get_comments;
+	core->print->hasrefs = (RPrintHasRefs)r_core_anal_hasrefs;
```

Comment:
😱

Reply:
what?

Reply:
scary cast, copypasta error

Reply:
I see all functions are assigned this way... shall I remove the cast for all these function pointers?